worker-markup-core/src/test/java/com/github/cafdataprocessing/worker/markup/core/EmailSplitterTest.java

```diff
@@ -36,6 +37,47 @@
+                "\n" +
+                "***********\n" +
+                "EDRM Enron Email Data Set has been produced in EML, PST and NSF format by ZL Technologies, Inc. This Data Set is licensed under a Creative Commons Attribution 3.0 United States License <http://creativecommons.org/licenses/by/3.0/us/> . To provide attribution, please cite to \"ZL Technologies, Inc. (http://www.zlti.com).\"\n" +
+                "***********\n" +
+                "\n";
```

Comment:
This string should be put into a test resource (i.e. in `/src/test/resources`) and read in using an `InputStreamReader` or `IOUtils.toString` or something like that, rather than being an escaped string in the code.

We can ignore since it looks like there are already plenty of other strings in other tests in this file which should also be test resources.